ilc/server/tailor/configs-injector.js

```diff
@@ -11,6 +11,7 @@ module.exports = class ConfigsInjector {
     #cdnUrl;
     #jsInjectionPlaceholder = '<!-- ILC_JS -->';
     #cssInjectionPlaceholder = '<!-- ILC_CSS -->';
+    #markedProdTags = /<!-- Prod only start -->.*?<!-- Prod only end -->/gims;
```

Comment:
what is gims ?

Reply:
does not it possible to handle on tailor-x level ?

Reply:
As we discussed with @b1ff - it is better to do LDE part removal on the ILC side

Reply:
> what is gims ?

This is for capturing correctly marked tags: include across multiple lines (s flag), The g flag ensures that it finds all occurrences in the input, the i flag ensures that it is case-insensitive, and the m flag ensures that the regular expression respects line breaks when matching the start or end of lines.